ext/tokenizer/tokenizer_data_gen.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+$infile = 'Zend/zend_language_parser.h';
+$outfile = 'ext/tokenizer/tokenizer_data.c';
+
+if (!file_exists($infile)) {
+    fwrite(STDERR, <<<ERROR
+    $infile is missing.
+    
+    Please, generate the PHP parser files by scripts/dev/genfiles
+    or by running the ./configure build step.
+    ERROR);
```

Comment:
Remove indentation here to keep this compatible with PHP < 7.3. We should probably bump our PHP version requirement, but this heredoc block probably isn't the right reason to do it :)